src/transformers/testing_utils.py

```diff
@@ -1292,6 +1293,16 @@ def require_jumanpp(test_case):
     return unittest.skipUnless(is_jumanpp_available(), "test requires jumanpp")(test_case)
 
 
+def require_python_311(test_case):
```

Comment:
Do we need the decorator? The test passes on Python < 3.11 anyway, so we can simplify the codebase and run it regardless!